app/src/main/java/com/quran/labs/androidquran/ui/fragment/AboutFragment.kt

```diff
@@ -0,0 +1,56 @@
+package com.quran.labs.androidquran.ui.fragment
+
+import com.quran.labs.androidquran.R
+
+import android.annotation.SuppressLint
+import android.os.Build
+import android.os.Bundle
+import android.view.View
+import android.view.ViewGroup
+import androidx.preference.*
+import androidx.recyclerview.widget.RecyclerView
+import com.quran.labs.androidquran.BuildConfig
+
+class AboutFragment : PreferenceFragmentCompat() {
+
+  override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
+    addPreferencesFromResource(R.xml.about)
+
+    val flavor = BuildConfig.FLAVOR + "Images"
+    val parent = findPreference("aboutDataSources") as PreferenceCategory
+    sImagePrefKeys.filter { it != flavor }.map {
+      parent.removePreference(findPreference(it))
+    }
+  }
+
+  // TODO: remove this function when issue https://issuetracker.google.com/issues/111662669 solved/released
+  override fun onCreateAdapter(preferenceScreen: PreferenceScreen?): RecyclerView.Adapter<*> {
```

Comment:
looks like they fixed it in 1.1.0-alpha01 - may you test it and see and let's kill this method if so?